frontend/appflowy_flutter/lib/workspace/presentation/command_palette/widgets/recent_views_list.dart

```diff
@@ -40,128 +39,115 @@ class RecentViewsList extends StatelessWidget {
           final showAskingAI = workspaceState?.userProfile.workspaceType ==
               WorkspaceTypePB.ServerW;
           final hoveredView = state.hoveredView;
-          return Column(
-            crossAxisAlignment: CrossAxisAlignment.start,
-            mainAxisSize: MainAxisSize.min,
-            children: [
-              AFDivider(),
-              Flexible(
-                child: Row(
-                  children: [
-                    Flexible(
-                      flex: 2,
-                      child: Align(
-                        alignment: Alignment.topLeft,
-                        child: ScrollControllerBuilder(
-                          builder: (context, controller) {
-                            return FlowyScrollbar(
+          return LayoutBuilder(
```

Comment:
Separate the widget tree into smaller widgets or a widget builder.

Reply:
fixed